### PR TITLE
Migrated test for host and server system properties

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

**Comment by honza-kasik** [comment]
Do not merge yet please, I forgot to edit one test

**Comment by honza-kasik** [comment]
Ready to be merged

File: common/src/main/java/org/jboss/hal/testsuite/page/config/SecurityPage.java [hunk 1]
```diff
@@ -137,15 +137,6 @@ public Boolean isErrorShownInForm() {
         return isElementVisible(selector);
     }
 
-    private Boolean isElementVisible(By selector) {
-        try {
-            Graphene.waitAjax().until().element(selector).is().visible();
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/SecurityPage.java:142]: Please remove no more needed `import org.jboss.arquillian.graphene.Graphene;`

Comment (honza-kasik) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/SecurityPage.java:142]: Removed import

**Comment by zebrik** [comment]
Thank you.

**Pull request merged**